tests/functional/prometheous/test_prometheus_validations.py

```diff
@@ -974,3 +975,81 @@ def test_restart_prometheus_glusterfs_pod(self):
             # Try to fetch metric from prometheus pod
             self._fetch_metric_from_promtheus_pod(
                 metric='heketi_device_brick_count')
+
+    @pytest.mark.tier2
+    def test_metrics_workload_on_prometheus(self):
+        """Validate metrics workload on prometheus"""
+
+        # Skip test if the prometheus pods are not present
+        openshift_ops.switch_oc_project(
```

Comment:
Add a cleanup for the project switch

Reply:
Cleanup to switch to self._registry_project ?  At line 992 ?

Reply:
You can first get the current project where currently it is.
Add it after 985

Reply:
After discussing with @SushilG96 ; it was agreed upon that it is not required here, since the method self._fetch_metric_from_promtheus_pod() called at Line 1016 already does it.